### DISCUSSION TITLE
Can the IMessageService and the IToastService be integrated?

### DISCUSSION THREAD

**Discussion by DefinitelyADev** [microsoft/fluentui-blazor#2416] [Q&A]
For example I would like to display a toast of every notification received. Though the `OnMessageItemsUpdated` doesn't provide me with the messages. Is there an obvious way to do it?

**Comment by vnbaaij** [comment]
Toasts and Messages are two different things:

>A message bar communicates important information about the state of the entire product or the surface where it appears, such as a page, drawer, dialog, or card

>A toast communicates the status of an action someone is trying to take or that something happened elsewhere in the app. Toasts are temporary surfaces. Use them for information that's useful and relevant, but not critical.

These are not interchangeable.

**Reply by DefinitelyADev** [reply]
I thought about it and I will probably end up creating a _proxy_ service that publishes to both.
